lib/pluginmanager/util.rb

```diff
@@ -1,12 +1,15 @@
 require "rubygems/package"
 
 module LogStash::PluginManager
+
   # check for valid logstash plugin gem name & version or .gem file, logs errors to $stdout
   # uses Rubygems API and will remotely validated agains the current Gem.sources
   # @param plugin [String] plugin name or .gem file path
   # @param version [String] gem version requirement string
+  # @param options [Hash] Configuration options used to setup external components
```

Comment:
actually, the proper way to documents options hash is like this, as described in http://www.rubydoc.info/gems/yard/file/docs/Tags.md#option 

```ruby
# @param [Hash] options the options used to setup external components
# @option options [Array<String>] :rubygems_source Gem sources to lookup for the verification
```